plugins/tracker-resources/src/types.ts

```diff
@@ -98,7 +98,7 @@ export const defaultPriorities = [
 
 export const issuesGroupBySorting: Record<IssuesGrouping, SortingQuery<Issue>> = {
   [IssuesGrouping.Status]: { '$lookup.status.rank': SortingOrder.Ascending },
-  [IssuesGrouping.Assignee]: { '$lookup.assignee.name': SortingOrder.Ascending },
+  [IssuesGrouping.Assignee]: { assignee: SortingOrder.Ascending },
```

Comment:
We lost name sorting, sad...

Reply:
Added issue: https://front.hc.engineering/workbench/platform/tracker/TSK-1344